keyboards/basekeys/slice/rev1_rgb/keymaps/via/keymap.c

```diff
@@ -0,0 +1,163 @@
+/* Copyright 2020 2Moons
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+#include "split_util.h"
```

Comment:
I don't think this is needed.

Reply:
keymap_jp isn't needed. 

Without QMK_KEYBOARD_H I can't find a workaround. Many Errors occur. 
split_util.h is for oled rotation. 
I've removed they JP Keymap.